src/main/res/layout/layout_authentication.xml

```diff
@@ -17,7 +17,9 @@
   ~ You should have received a copy of the GNU General Public License
   ~ along with Amahi. If not, see <http ://www.gnu.org/licenses/>.
   -->
-
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+	android:layout_width="fill_parent"
+	android:layout_height="fill_parent">
```

Comment:
Make both of these `match_parent` instead of `fill_parent`. As both are same but `fill_parent` is deprecated and was replaced by `match_parent` after api level 8.
Read [here](https://developer.android.com/reference/android/view/ViewGroup.LayoutParams.html#FILL_PARENT).